src/main/java/ch/heigvd/amt/stack/infrastructure/persistence/remote/RemoteGamificationRepository.java

```diff
@@ -22,21 +28,88 @@
+      new Badge().name("QBadge2").category("questions").title("Wizard of questions")
+          .description("Second questions badge").pointsLower(29).pointsUpper(100),
+      new Badge().name("QBadge3").category("questions").title("Grand wizard of questions")
+          .description("Third questions badge").pointsLower(99).pointsUpper(1000),
+      new Badge().name("QBadge4").category("questions").title("Very grand wizard of questions")
+          .description("Fourth questions badge").pointsLower(999),
+
+      new Badge().name("ABadge1").category("answers").title("Apprentice of answers")
+          .description("First answers badge").pointsLower(0).pointsUpper(30),
+      new Badge().name("ABadge2").category("answers").title("Maester of answers")
+          .description("Second answers badge").pointsLower(29).pointsUpper(100),
+      new Badge().name("ABadge3").category("answers").title("Archimaester of answers")
+          .description("Third answers badge").pointsLower(99).pointsUpper(1000),
+      new Badge().name("ABadge4").category("answers").title("Grand maester of answers")
+          .description("Fourth answers badge").pointsLower(999),
+
+      new Badge().name("CBadge1").category("comments").title("Beast of comments")
+          .description("First comments badge").pointsLower(0).pointsUpper(30),
+      new Badge().name("CBadge2").category("comments").title("Monster of comments")
+          .description("Second comments badge").pointsLower(29).pointsUpper(100),
+      new Badge().name("CBadge3").category("comments").title("Big monster of comments")
+          .description("Third comments badge").pointsLower(99).pointsUpper(1000),
+      new Badge().name("CBadge4").category("comments").title("Terrifying monster of comments")
+          .description("Fourth comments badge").pointsLower(999)
+  ));
 
   private RemoteGamificationRepository() {
     Configuration.getDefaultApiClient().setApiKey(System.getenv("GAMIFY_API_TOKEN"));
     Configuration.getDefaultApiClient().setBasePath(System.getenv("GAMIFY_SERVER"));
 
-    this.addCategory("questions", "Questions", "Addition of new questions");
-    this.addCategory("answers", "Answers", "Addition of new answers");
-    this.addCategory("comments", "Comments", "Addition of new comments and votes for them");
-
-    this.addRule("questionRule", "questions", GamificationEvent.NEW_QUESTION, 10);
-    this.addRule("answerRule", "answers", GamificationEvent.NEW_ANSWER, 30);
-    this.addRule("commentRule", "comments", GamificationEvent.NEW_COMMENT, 20);
-    this.addRule("upvoteRule", "comments", GamificationEvent.UPVOTE, 10);
-    this.addRule("downvoteRule", "comments", GamificationEvent.DOWNVOTE, 0);
-    this.addRule("acceptAnswerRule", "answers", GamificationEvent.SELECTION, 25);
+    for (var c : categories) {
+      this.addCategory(c);
+    }
+    for (var c : rules) {
+      this.addRule(c);
+    }
+    for (var b : badges) {
+      this.addBadge(b);
+    }
```

Comment:
Not really important but it's cleaner IMO
```suggestion
    categories.forEach(this::addCategory);
    rules.forEach(this::addRule);
    badges.forEach(this::addBadge);
```